_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+A dashboard combines multiple data visualizations into a single view. With the **Dashboard** application, you can:
+
+- Get started with pre-built panels and dashboards and customize display properties to create the dashboard you need.
+- Analyze, monitor, and alarm on metrics, logs, and traces across multiple data sources.
```

Comment:
This seems out of scope for a beginner-targeted tutorial, particularly because we haven't yet launched the anywhere project. I'd recommend removing.

Reply:
Good point. In thinking more braodly about dashboards capabilitites, I've revised this intro.